machine-learning/solutions/seismic_salt_detection/trainer/inputs.py

```diff
@@ -1,39 +1,39 @@
 #!/usr/bin/env python
-#Copyright 2018 Google LLC
+# Copyright 2018 Google LLC
 #
-#Licensed under the Apache License, Version 2.0 (the "License");
-#you may not use this file except in compliance with the License.
-#You may obtain a copy of the License at
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
-#Unless required by applicable law or agreed to in writing, software
-#distributed under the License is distributed on an "AS IS" BASIS,
-#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#See the License for the specific language governing permissions and
-#limitations under the License.
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 # ==============================================================================
 """Takes care of all the input data for the model.
 
-  Parses columns and generates input specifications for `Estimator`.
+Parses columns and generates input specifications for `Estimator`.
 
-  Typical usage example:
+Typical usage example:
 
-  inputs.get_train_spec(parameters)
-  inputs.get_eval_spec(parameters)
+inputs.get_train_spec(parameters)
```

Comment:
I don't think this usage is quite "typical" -- the return values aren't even saved.  Could you please either:

* add a couple more lines of code for what one is supposed to do with the artifacts generated by these functions
or 
* describe in English what the important functions are and how to use them?

Reply:
will fix

Reply:
good!